### PR TITLE
Installing a Custom Linux Distribution on a KVM Linode

### PR DESCRIPTION
**Pull request by thetechstewart**
This is an import from a submission given to us via email. It has not yet been reviewed for technical accuracy or edited for for proper formatting. It's purpose here is to gauge interest from the community on this subject, and solicit independent review of the contents.

### CODE REVIEW

**Comment by FelicianoTech** [comment]
Ready for copy edit.

File: docs/tools-reference/custom-kernels-distros/custom-distro-kvm.md [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+- Under the **Console Access** header, click on the **Launch Lish Ajax Console** link.
+
+	[![Launch Lish](/docs/assets/snap0007.png)](/docs/assets/snap0007.png)
+- Enter the following commands into the Lish Web Console:
+	1. mkdir /mnt/system	
+	2. mount /dev/sdb /mnt/system
+	3. cd /mnt/system
+	4. wget http://less.cogeco.net/CentOS/6.6/isos/x86_64/CentOS-6.6-x86_64-netinstall.iso
+		> Alternate mirrors: http://www.centos.org/download/mirrors/
+	5. dd if= CentOS-6.6-x86_64-netinstall.iso of=/dev/sda
```
Comment (FelicianoTech) [inline docs/tools-reference/custom-kernels-distros/custom-distro-kvm.md]: Should be `dd if=CentOS-6.6-x86_64-netinstall.iso of=/dev/sda`

There's a misplaced space character before the ISO filename.

**Comment by thetechstewart** [comment]
Revised with fix noted from tech edit.

**Comment by alexfornuto** [comment]
The big issue with this guide is that the installation method creates a partition system on the disk, which breaks the disk resize functionality, and prevents the user from using the backups service. That doesn't mean I don't want to proceed, we just need to make sure that's made clear, and offer another guide for custom distros on KVM Linodes that uses the other method of copying disk images up from a virtual image.

My current changes are here: https://github.com/thetechstewart/docs/pull/2. Don't merge yet, because I'm not done; I just wanted to put up what I have before leaving for the weekend :smile:

**Comment by alexfornuto** [comment]
I replied to the author over email with the concerns noted above, as well as the inclusion of Glish to make the guide more distro-agnostic.

**Comment by ellekrout** [comment]
This guide has gone a month without author communication. I'm closing out this PR. Should the author return (or you are the author), please open a new PR for the guide.

**Pull request closed without merging**